src/style.css

```
@@ -0,0 +1,173 @@
+html {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 198px;
+  height: 404px;
```

you haven't fixed the width after applying `box-sizing: border-box;`
it should be 200px
and you don't need height measure - proper contents will dictate the height
```suggestion
  width: 200px;
```